### PR TITLE
Megaman - ZLL RGBW Bulb

### PR DESCRIPTION
**Pull request by Duncankn**
Submitting 'DevWs' for publication

### CODE REVIEW

**Comment by shrakuma** [comment]
Hi @Duncankn 
Thank you for working on integrating with our platform. We are reviewing the code and we shall keep you posted here and in the other PRs that you have submitted.

**Comment by shrakuma** [comment]
@bflorian can you please re-open this PR.

**Comment by MAblewicz** [comment]
Reopened by Shravan's request.

File: devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy [hunk 1]
```diff
@@ -52,6 +52,7 @@ metadata {
         fingerprint profileId: "C05E", inClusters: "0000,0003,0004,0005,0006,0008,0300,1000", outClusters: "0019", manufacturer: "Philips", model: "LST002", deviceJoinName: "Philips Hue Lightstrip"
         fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "innr", model: "RB 185 C", deviceJoinName: "Innr Smart Bulb Color"
         fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "innr", model: "FL 130 C", deviceJoinName: "Innr Flex Light Color"
+        fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FFFF", outClusters: "0019",manufacturer: "Megaman", model: "ZLL-ExtendedColor", deviceJoinName: "INGENIUM ZB RGBW Light"
```
Comment (greens) [inline devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy]: missing a space before `manufacturer`. if this was auto-generated it needs to be addressed with the dev workspace team

Comment (Duncankn) [reply devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy]: A space before manufacturer was added

Comment (Duncankn) [reply devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy]: @greens I could not re-submit the PR and always shown "There was an error submitting your request" on smartThings IDE.
Please advice, thanks!

Comment (Duncankn) [reply devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy]: @greens this issue resolved, thanks!

**Review by greens — APPROVED** [review]

**Pull request merged**